codespell_lib/data/dictionary_code.txt

```diff
@@ -65,5 +65,6 @@ udate->update, date,
 udates->updates, dates,
 uint->unit
 uis->is, use,
+usesd->used
```

Comment:
```suggestion
usesd->used, uses,
```

As suggested by @luzpaz in https://github.com/codespell-project/codespell/pull/2331#issuecomment-1092362357